src/ui/components/ToggleOnMobileCard/index.js

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import Card from 'ui/components/Card';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+
+export class ToggleOnMobileCardBase extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    header: PropTypes.node,
+    i18n: PropTypes.object.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { expanded: false };
+  }
+
+  onClick = (event) => {
+    event.preventDefault();
+
+    this.setState({ expanded: !this.state.expanded });
```

Comment:
It would be nicer to use Redux here but I guess you'd have to generate an ID [like the ErrorHandler](https://github.com/mozilla/addons-frontend/blob/master/src/core/errorHandler.js#L97) and this patch is already massive so no biggie.